DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -418,5 +418,15 @@ public void ActivateDelegateUser_activates_delegate_user()
             // Then
             userDataService.GetDelegateUserById(delegateUserId)!.Active.Should().BeTrue();
         }
+
+        [Test]
+        public void AddDelegateToGroup_get_delegates_not_registered_for_group_by_groupId()
+        {
+                // When
+                var result = userDataService.GetDelegatesNotRegisteredForGroupByGroupId(5, 101);
+                    
+                // Then
+                result.Count.Should().Be(30);

```

Comment:
Can use `result.Should().HaveCount(30)` here.

Reply:
I have updated as suggested